src/SignalR/perf/benchmarkapps/Crankier/Server/Startup.cs

```diff
@@ -11,15 +12,23 @@ namespace Microsoft.AspNetCore.SignalR.Crankier.Server
     public class Startup
     {
         private readonly IConfiguration _config;
+        private readonly string _azureSignalrConnectionString;
         public Startup(IConfiguration configuration)
         {
             _config = configuration;
+            _azureSignalrConnectionString = configuration.GetSection("Azure:SignalR").GetValue<string>("ConnectionString", null);
+            if (_azureSignalrConnectionString != null)
+                Console.WriteLine("Using Azure SignalR");
```

Comment:
nit: if brackets

It's probably better to put this WriteLine in ServerCommand.Execute anyway.